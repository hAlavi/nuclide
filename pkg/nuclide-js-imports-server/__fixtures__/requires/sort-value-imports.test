import 'ccc-side-effect';
import AAA from 'AAA';
import bbb from 'bbb';


import 'bbb-side-effect';
import CCC from 'CCC';
import aaa from 'aaa';


import 'aaa-side-effect';
import BBB from 'BBB';
import ccc from 'ccc';

// Make sure they are used.
aaa(AAA);
bbb(BBB);
ccc(CCC);
