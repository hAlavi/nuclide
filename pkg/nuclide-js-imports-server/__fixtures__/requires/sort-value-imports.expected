import 'aaa-side-effect';
import 'bbb-side-effect';
import 'ccc-side-effect';

import AAA from 'AAA';
import BBB from 'BBB';
import CCC from 'CCC';

import aaa from 'aaa';
import bbb from 'bbb';
import ccc from 'ccc';

// Make sure they are used.
aaa(AAA);
bbb(BBB);
ccc(CCC);
